src/cogent3/core/new_sequence.py

```diff
@@ -2425,40 +2460,40 @@ def _seq_filter(self, seq):
 
 
 @singledispatch
-def _coerce_to_seqview(data, seqid):
+def _coerce_to_seqview(data, seqid, alphabet):
```

Comment:
add comment about having SeqViewABC as return type

Reply:
have just added the return type